lib/kernel/doc/src/pg.xml

```diff
@@ -197,34 +324,71 @@
     </func>
 
     <func>
-      <name name="get_local_members" arity="1" since="OTP 23.0"/>
-      <name name="get_local_members" arity="2" since="OTP 23.0"/>
-      <fsummary>Return all local processes in a group.</fsummary>
+      <name name="get_members" arity="1" since="OTP 23.0"/>
+      <name name="get_members" arity="2" since="OTP 23.0"/>
+      <name name="get_members" arity="3" since="OTP 27.0"/>
+      <fsummary>Return all members in a group.</fsummary>
       <desc>
-        <p>Returns all processes running on the local node in the
-          group <c>Group</c>. Processes are returned in no specific order.
-          This function is optimised for speed.
-        </p>
+        <p>If <c>Shard</c> is not specified, returns all members in the
+          singleton group <c>Group</c>. If <c>Shard</c> is specified,
+          returns all members in the <c>Shard</c> of sharded group
+          <c>Group</c>. Members are returned in no specific order.
+          This function is optimised for speed.</p>
       </desc>
     </func>
 
     <func>
-      <name name="get_members" arity="1" since="OTP 23.0"/>
-      <name name="get_members" arity="2" since="OTP 23.0"/>
-      <fsummary>Return all processes in a group.</fsummary>
+      <name name="get_local_members" arity="1" since="OTP 23.0"/>
+      <name name="get_local_members" arity="2" since="OTP 23.0"/>
+      <name name="get_local_members" arity="3" since="OTP 27.0"/>
+      <fsummary>Return all local members in a group.</fsummary>
       <desc>
-        <p>Returns all processes in the group <c>Group</c>.
-          Processes are returned in no specific order.
-          This function is optimised for speed.</p>
+        <p>If <c>Shard</c> is not specified, returns all members running on the
+          local node in the singleton group <c>Group</c>. If <c>Shard</c> is
+          specified, returns all members running on the local node in the
+          <c>Shard</c> of sharded group <c>Group</c>. Members are returned in
+          no specific order. This function is optimised for speed.</p>
       </desc>
     </func>
 
```

Comment:
What happened to the `get_metadatas()` functions? No way to get the metadata now?

Reply:
I'm thinking of these options and mark get_members as deprecated later:
which_members
members
all_members

I'm thinking of other names like these but it will also confuse people and make then think these will exclude Pid members but only have tuple members.
get_members_with_metadata
get_metadata_members

so far `members` is the best I like